src/app/WriteHandler.h

```diff
@@ -69,6 +71,7 @@ class WriteHandler : public Messaging::ExchangeDelegate
      *  construction until a call to Close is made to terminate the
      *  instance.
      *
+     *  @param[in] apProvider              A Valid pointer to the model used to forward writes towards
```

Comment:
```suggestion
     *  @param[in] apProvider              A valid pointer to the model used to forward writes towards
```
Yes, the thing you copy/pasted has the same problem.